python/Quickstart.MD

```diff
@@ -142,192 +119,201 @@ Body of event request:
-
->>> event_data = {"eventName": "agile-process-teams:add-direct-supplier-incident:v2",
-                  "ownerId": "94f94f37-2772-4b39-8041-9c2dcfcfff82",
-                  "processNetworkId": "945feb8e-09a7-4fef-9a8a-b3c5b56f87d"}
->>> request_headers = utils.create_headers(my_token)
->>> payload_data = utils.read_payload_file(my_file)
->>> request_body = utils.create_payload(event_data, payload_data)
->>> result = utils.post_request(request_headers, request_body)
+import requests
+url = "URL_FROM_YOUR_ADMINISTRATOR"
+result = requests.request("POST", url, headers=request_header, data=request_body_json)
 ```
 An example successful response:
 ```
 {
     "header": {
         "headerVersion": 1,
-        "eventName": "agile-process-teams:add-direct-supplier-incident-response:v2",
-        "ownerId": "94f94f37-2772-4b39-8041-9c2dcfcfff82",
+        "eventName": "agile-process-teams:add-direct-supplier-incident-response:v3",
+        "ownerId": "YOUR_OWNER_ID",
         "isErr": false,
         "errCode": "200_OK",
-        "licensePlate": "qL0BJs"
+        "licensePlate": "LICENSE_PLATE_ID"
     },
     "payload": {
-        "id": "fd04dd70-c162-43ef-8118-539ca01de471"
+        "id": "ITEM_ID"
     }
 }
 ```
 
 ## Response Handling
 
-The `id` in a successful response payload is the identifier of the created incident. Notice that the response is a string, not a JSON object. To get this identifier, enter the following in your Python terminal:
-```
->>> import json
->>> response_json = json.loads(result.text)
->>> incident_id = response_json.get('payload').get('id')
-```
+The `id` in a successful response payload is the identifier of the created incident. Notice that the response is a JSON object that has been converted to a string for transmission. To work with this data, convert it to a Python dictionary.  
 
-To ensure your value stored properly, enter the following in your Python terminal:
+**Type or copy the following into your Python terminal**  
 ```
->>> incident_id
-'fd04dd70-c162-43ef-8118-539ca01de471'
+response_json = json.loads(result.text)
+incident_id = response_json.get('payload').get('id')
 ```
 
+You may now reference your created incident id by using its variable name.
+
 ## Add Incident Comment
 
-Prepare a comment payload file called `comment_payload.json` containing the below:
+The APT-SCWM shared API contains processes for adding, editing, or listing comments for incidents along with other utility functions.  The header of this request body is slightly different (in this case only the `eventName` changes) and uses the same authorization configuration as the previous request.  
+If you are building or modifying your requests programmatically, you could use the `ITEM_ID` you created in the previous step.  The JSON library allows us to use `json.loads(string)` to create a dictionary from an existing JSON object and then use the `json.dumps(dictionary)` to convert our modified dictionary back to a JSON object.  
```

Comment:
"If you are building or modifying your requests programmatically, use the `ITEM_ID` created in the previous step. The JSON library allows using `json.loads(string)` to create a dictionary from an existing JSON object, followed by `json.dumps(dictionary)` to convert the modified dictionary back to a JSON object."

Reply:
reworded